operations/structure.md

```diff
@@ -0,0 +1,45 @@
+# Structure and responsibilities
+
+This page describes the structure and roles of our Product Team.
+
+## Responsibilities
+
+2i2c's Operatoins team is responsible for defining and overseeing our systems for work coordination, planning, and accountability.
+
+Below are major areas of responsibility.
+
+### Define systems and processes for coordination
+
+- Define our meeting and coordination systems for planning our work.
+- Define systems for sharing information between team members.
+
+### Refine work items to be actionable and clear
+
+- Translate strategic priorities and deliverables into actionable work items.
+- Ensure that our work items have a clear scope and path forward for completion.
+- Communicate with our Product and Engineering teams to align our teams on a plan.
+
+### Oversee work for projects and teams
+
+- Assign work items to team members on a given project according to team priorities.
+- Hold team members accountable for completing work on-time, or for adjusting scope as needed.
+
+### Communicate progress and status to key stakeholders
+
+- Make our work transparent, discoverable, and understanding to internal and external stakeholders.
+- Communicate with internal and external stakeholders to share information about the status and progress of major work items.
+- Surface changes in scope or identified risk in accomplishing our goals.
+
+## Team structure
+
+Our operations team largely consists of **Project Managers** that oversee the coordination and execution of work on a given project.
+They tend to **oversee backlogs** that a number of team members work on, and work with key stakeholders to **ensure that our teams execute their work** according to our priorities and objectives.
+
+## Team roles
+
+Project Manager
+: Ensures that a team plans and executes its work in a way that aligns with the team's objectives. See above for specific responsibilities and actions.
+
+## Membership
+
+Damian Avila currently serves as a part-time Project Manager for our Engineering and Product team.
```

Comment:
After thinking more about this, I actually like a lot to see an operations area in the structure.
Can you "formally" add this area to the escalidraw organizational diagram?

Reply:
I agree it should live under an operation branch/area in the future... for now, I guess we could have it floating between the second and third layer, to the right, maybe? Do we need actually a 3D escalidraw representation 😜 ???